<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite thread-count="2" parallel="tests" name="Failed suite [Outage Testing]" guice-stage="DEVELOPMENT">
  <test thread-count="2" parallel="tests" name="2(failed)">
    <classes>
      <class name="outagetesting.ParallelOutageTesting">
        <methods>
          <include name="usertest2"/>
          <include name="usertest1"/>
          <include name="setUp"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- outagetesting.ParallelOutageTesting -->
    </classes>
  </test> <!-- 2(failed) -->
  <test thread-count="2" parallel="tests" name="1(failed)">
    <classes>
      <class name="outagetesting.ParallelOutageTesting">
        <methods>
          <include name="usertest2"/>
          <include name="usertest1"/>
          <include name="setUp"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- outagetesting.ParallelOutageTesting -->
    </classes>
  </test> <!-- 1(failed) -->
</suite> <!-- Failed suite [Outage Testing] -->
